adam-core/src/main/scala/org/bdgenomics/adam/algorithms/consensus/ConsensusGenerator.scala

```diff
@@ -54,7 +54,7 @@ object ConsensusGenerator {
    *   present in a single aligned read back into the reference sequence where
    *   they are aligned.
    */
-  def fromReads: ConsensusGenerator = {
+  def fromReads(): ConsensusGenerator = {
```

Comment:
If we want it to be (easily) callable from Java (and thus Python), it seems that we need to have the parens.